### DISCUSSION TITLE
좋은 글과 좋은 코드에 대하여

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#27] [4. 기타 글 모음]
저의 첫 Discussion 이네요 🎉

[좋은 글과 좋은 코드는 닮았다.](https://velog.io/@shkisme/%EC%A2%8B%EC%9D%80-%EA%B8%80%EA%B3%BC-%EC%A2%8B%EC%9D%80-%EC%BD%94%EB%93%9C%EB%8A%94-%EB%8B%AE%EC%95%98%EB%8B%A4)

최근에 자주 하고 있는 좋은 글과 좋은 코드에 대한 제 생각을 짧은 글로 써봤습니다. 생각보다 공통점이 훨씬 더 많은 것 같아요!

원래는 `이펙티브 자바` 책의 내용을 정리한 글을 꾸준히 업로드 하려고 했었는데, 생각해보니 책 내용을 그대로 정리만 한 글이 의미가 있나? 싶더라구요! 물론, 정리를 하며 공부를 하고, 다시 정리글을 읽으며 복습하시는 분들에게는 도움이 될 것 같아요 ㅎㅎ 그런데 저는 그런 제 글을 잘 안 읽게 되더라구요.

앞으로 제 기술 블로그에는 제가 습득한 내용을 바탕으로 응용해보고, 직접 공들여 만든 글을 업로드 하려고 해요! 아직은 글쓰기에도 미숙하기에 쉽지 않은 길일 것 같네요... ㅎㅎ

꾸글쓰 챌린지 화이팅!

+) 종강 하고싶다ㅏㅏㅏㅏ

**Comment by pnudpwls1103** [comment]
꾸준히 코드 짜고 글 써야 하는데 습관 만들기 참 어려운거 같습니다 ㅠㅠ 
좋은 습관을 들이기 위해 노력해봐야겠어요.

(나도 설계 잘 하고 싶당.. 코드도 이쁘게 짜고 싶당!!!)
